core/src/main/scala/dagr/core/config/Configuration.scala

```diff
@@ -104,124 +51,19 @@ private[core] object Configuration extends ConfigurationLike {
   * that is then mixed into the Configuration object above, and used to create a concrete
   * Configuration trait below.
   */
-private[config] trait ConfigurationLike extends LazyLogging {
-  private[config] def config : Config
+private[config] trait ConfigurationLike extends LazyLogging with dagr.sopt.util.ConfigurationLike {
```

Comment:
I really don't see the point of this separate `ConfigurationLike` trait, and then the separate `Configuration` trait below - I think it makes everything more confusing.  Why not just have `Configuration` traits both here and in `sopt`, and have this `Configuration` trait extend `sopt.Configuration`?  I see no added value in this separation.